packages/components/src/dialog/button-bar.tsx

```diff
@@ -15,10 +15,8 @@ export type BaseButton = {
 	target?: string;
 };
 
-export type Button = ReactElement | BaseButton;
```

Comment:
I suppose this isn't strictly necessary any more, but I didn't love having yet another plain `Button` type that isn't very meaningful, so I just inlined the types in the few places it was used.